CRM/Contact/Form/DedupeFind.php

```diff
@@ -41,9 +41,11 @@ public function buildQuickForm() {
     $groupList = ['' => ts('- All Contacts -')] + CRM_Core_PseudoConstant::nestedGroup();
 
     $this->add('select', 'group_id', ts('Select Group'), $groupList, FALSE, ['class' => 'crm-select2 huge']);
-    if (Civi::settings()->get('dedupe_default_limit')) {
-      $this->add('text', 'limit', ts('No of contacts to find matches for '));
-    }
+    $this->add('text', 'limit', ts('No of contacts to find matches for '));
```

Comment:
It's a pre-existing condition, but this text makes no sense to me, and trailing spaces are not allowed within `ts()`

Reply:
@colemanw I do agree, but I'm struggling to think of a better string (other than writing "number" in full). Maybe a `{help}` popup is required to add some longer-form description?